### DISCUSSION TITLE
What other achievements other than besides YOLO, Pull shark, and Galaxy Brain?

### DISCUSSION THREAD

**Discussion by imanoar** [tarikmanoar/tarikmanoar#3] [Q&A] [answered]
### Please help me

**Comment by tarikmanoar** [comment, marked as answer]
If you are referring to the new **achievements**, other than those 3, we do not know what other achievements there are (at the moment).

If you are referring to the old achievements, there's Arctic Code Vault Contributor, Public Sponsor, and Mars 2020 Contributor. Only Public Sponsor is available to get now.
